File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(int a, int b) {
    string as = to_string(a), bs = to_string(b);
    int answer = max(stoi(as+bs), stoi(bs+as));
    return answer;
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        int c = Integer.valueOf(Integer.toString(a) + Integer.toString(b));
        int d = Integer.valueOf(Integer.toString(b) + Integer.toString(a));
    
        if(Integer.valueOf(c) > Integer.valueOf(d)){
            answer = c; 
        } else {
            answer = d;
        }
        return answer;
    }
}